fcrepo-persistence-api/src/main/java/org/fcrepo/persistence/api/CommitOption.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.api;
+
+/**
+ * This option enum specifies how commits should be applied to the PersistentStorageSession.
+ *
+ * @author dbernstein
+ */
+public enum CommitOption {
+    NEW_VERSION,  //indicates that you want to commit changes to a new version
```

Comment:
My understanding was that `PersistentStorageSession.commit` committed changes for one to many objects, but it seems like whether changes were committed to a new version or unversioned would vary by object?

Reply:
I would think that from the http layer's point of view they are either committing to the HEAD or they are wanting to create a new version.   The HTTP layer is not really aware of what OCFL objects are being touched, no?